app/gui/view/graph-editor/src/lib.rs

```diff
@@ -1428,6 +1430,20 @@ impl GraphEditorModelWithNetwork {
         node_id
     }
 
+    /// Move node to the `edited_node` scene layer, so that it is rendered by the separate camera.
+    pub fn move_node_to_edited_node_layer(&self, node_id: NodeId) {
```

Comment:
Since this and the following method operate only on the `nodes`, they might be better suited to live in the `Nodes` struct directly.